_component_design/help-beacon.md

```diff
@@ -14,31 +14,61 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
-The beacon can be used to identify the location of new functionality or
-important features for first-time user.
 
--   Place the beacon to the right or left of navigation items, labels or
-    section headers.
+## Introduction
 
--   Use one beacon at a time.
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
--   Remove the beacon once it has been viewed.
+### When to use
 
--   Avoid using the beacon for warnings or alerts.
+A beacon is best used in the following scenarios:
 
--   Don't obsure text with a beacon.
+- Show when new products/features have been added to the control panel that improve existing functionality.
+- Show new users the location of commonly used features to aid them in getting started.
+- **Do not** use multiple beacons at one time.
```

Comment:
Create a new list as these last two bullets don't match the introductory text:
```markdown
There are some cases where beacons should not be used:

- Do **not** use multiple beacons at one time.
- Do **not** use the beacon to alert users of warnings or alerts.
```

Reply:
updated.